internal/pkg/coordinator/monitor.go

```diff
@@ -157,6 +158,7 @@ func (m *monitorT) handlePolicies(ctx context.Context, hits []es.HitT) error {
 		var policy model.Policy
 		err := hit.Unmarshal(&policy)
 		if err != nil {
+			m.log.Debug().Err(err).Msg("Failed to deserialize policy json")
```

Comment:
What is the reason you put this on Debug level? Do you expect this to happen often?